media-libs/libiptcdata/libiptcdata-1.0.5.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_7,3_8} )
+
+inherit eutils python-single-r1
```

Comment:
I don't see `eutils` in use anywhere here.

Reply:
indeed, removed